actions/websocket_actions.jsx

```diff
@@ -462,6 +462,11 @@ function handleDirectAddedEvent(msg) {
 function handleUserAddedEvent(msg) {
     if (ChannelStore.getCurrentId() === msg.broadcast.channel_id) {
         getChannelStats(ChannelStore.getCurrentId())(dispatch, getState);
+        dispatch({
+            type: UserTypes.RECEIVED_PROFILE_IN_CHANNEL,
+            data: {user_id: msg.data.user_id},
+            id: msg.broadcast.channel_id,
```

Comment:
Why is `id` not in `data.channel_id`?

Reply:
I observed that too but didn't bother to update since `id` in this dispatch signature is actually sometimes teamId or channelId.
So maybe, it will be better to put `id` inside the `data`?  Like `data: {id, user_id}` but I don't think worth it's updating, 0/5 on that.

Reply:
Sure, [MM-10455](https://mattermost.atlassian.net/browse/MM-10455) ticket created.
It's not a breaking change but is a missing feature I think. I'm just not sure how to better implement that without using that broadcast object.